xslt/findings2jira.xsl

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0"?>
```

Comment:
This file is no longer needed. A JIRA field is included in the main export

Reply:
I agree with @syrosh : @gronke , could you please remove this file from this PR? Thanks

Reply:
Commit removed.